### ISSUE TITLE
Unable to open asset

### ISSUE THREAD

**Issue by Zoha131** [kexanie/MathView#24]
When I run the app this error shows in log cat. However, the app runs well.
In the future is this error can create any fatal error? And what is the solution for this kind of error?

![capture2](https://cloud.githubusercontent.com/assets/16666716/17346188/6cef54ba-592b-11e6-968d-947c493387fa.PNG)

**Comment by Makman2** [comment]
Getting the same for the usage example in the readme:
```
01-02 03:05:19.688 13031-13153/de.tuhh.mst.oxid E/AndroidProtocolHandler: Unable to open asset URL: file:///android_asset/MathJax/images/CloseX-31.png?rev=2.5.3
01-02 03:05:20.639 13031-13119/de.tuhh.mst.oxid E/AndroidProtocolHandler: Unable to open asset URL: file:///android_asset/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.5.3
01-02 03:05:20.649 13031-13119/de.tuhh.mst.oxid E/AndroidProtocolHandler: Unable to open asset URL: file:///android_asset/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.5.3
01-02 03:05:20.669 13031-13119/de.tuhh.mst.oxid E/AndroidProtocolHandler: Unable to open asset URL: file:///android_asset/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.5.3
```

**Comment by ghost** [comment]
Use this link, and download the image. 
http://www.stat.washington.edu/~mgrffn/564/lab2_files/mathjax-23/images/CloseX-31.png
after downloading this image, paste it in your app/src/main/assets/MathJax/images, this will solve the image issue.

**Comment by ghost** [comment]
https://redbooth.com/d/03957cf9f7acf775
Download this, then goto this location /TeX/TeX/css and copy the woff folder and paste this in app/src/main/assets/MathJax/fonts/HTML-CSS/TeX
Then all your issues will be resolved.

**Comment by Makman2** [comment]
Those files should be included into MathView anyway^^

**Comment by ghost** [comment]
That is not included, please check your directory. In my case it was not included.

**Comment by Makman2** [comment]
Yeah I mean this issue is for including those files into MathView then :)

**Comment by ghost** [comment]
Right now, I'm having this issue, hope you can figure it out. 
**cr_BindingManager: Cannot call determinedVisibility() - never saw a connection for the pid: 18999**

Till now I tried all the links of stack-overflow. 
```
getSettings().setJavaScriptEnabled(true);
getSettings().setDomStorageEnabled(true);
   setWebViewClient(new WebViewClient() {
            public boolean shouldOverrideUrlLoading(WebView view, String url) {
                return false;
            }
        });

```
Still I'm seeing that in my logcat. Any Help.

**Comment by Makman2** [comment]
I think you should file a new issue for this one, looks like a different problem :+1:

**Comment by ghost** [comment]
This is something related to WebView...anyways, I'll create a new Issue.

**Comment by RoseAlb** [comment]
Crash happened because of this error
JNI DETECTED ERROR IN APPLICATION: JNI CallObject method called with pending exception
IllegalStateException: Unable to create layer for Mathview

**Comment by Makman2** [comment]
Looks like a totally different issue, this one is just about the warnings from above. Please file new issues to ease development and issue triaging for the maintainers (although the repo seems abandoned...).

**Comment by tonmoyray** [comment]
> Use this link, and download the image. http://www.stat.washington.edu/~mgrffn/564/lab2_files/mathjax-23/images/CloseX-31.png after downloading this image, paste it in your app/src/main/assets/MathJax/images, this will solve the image issue.

The link does not work !
